src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -644,6 +642,11 @@ def get_observation(self, event: Event) -> pd.DataFrame:
         )
 
         # not all simulants with a guardian are eligible to be dependents
+        # need to know earnings of dependents in past year for this
+        dependent_w2 = df_w2[df_w2["simulant_id"].isin(df_dependents.index.unique())]
+        dependent_wages = dependent_w2.groupby("simulant_id")["income"].sum()
+        df_dependents["last_year_income"] = df_dependents.index.map(dependent_wages)
```

Comment:
Couldn't this just be `= dependent_wages`?

Also, the use of "earnings," "income" and "wages" interchangeably is confusing. This variable could be called "dependent_last_year_income". The comment above should say "income".

What happens here when there is no income for the dependent? I'd expect NaN, which would cause them to not qualify below if they are over 19.

Reply:
all great points. I have attempted to address them in this next commit.